docs/administrators/operational_procedures.rst

```diff
@@ -140,6 +140,46 @@ Procedure
 
 8. If this model uses LSM, perform initial tests of all services via the API.
 
+Extra careful deploy procedure
++++++++++++++++++++++++++++++++
+
+For models that are considered risky, it is possible to enable the model in a more gradual way.
+The general idea is to disengage all features on the orchestrator that make the agents perform unsupervised deployments.
+Then the agents can be activated by hand, one-by-one.
+
+
+1. Take note of the following settings
+
+    * `autostart_agent_deploy_interval`
+    * `autostart_agent_repair_interval`
+
+2. Disable spontaneous deployment
+
+    * `autostart_agent_deploy_interval = 0`
+    * `autostart_agent_repair_interval = 0`
+    * `auto_deploy = True`
+    * `push_on_auto_deploy = False`
```

Comment:
This assumes that only autostarted agents are used. Maybe we should mention that explicitly.

Reply:
added